src/components/standalone/firewall/conntrack/ConntrackContent.vue

```diff
@@ -0,0 +1,198 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import { ubusCall } from '@/lib/standalone/ubus'
+import ConntrackRecordsTable from '@/components/standalone/firewall/conntrack/ConntrackRecordsTable.vue'
+import DeleteConntrackRecordModal from '@/components/standalone/firewall/conntrack/DeleteConntrackRecordModal.vue'
+import {
+  NeHeading,
+  NeInlineNotification,
+  NeButton,
+  NeSkeleton,
+  NeEmptyState,
+  NeTextInput
+} from '@nethesis/vue-components'
+import { getAxiosErrorMessage } from '@nethesis/vue-components'
+import { onMounted, ref, computed } from 'vue'
+import { CanceledError } from 'axios'
+
+const { t } = useI18n()
+
+export type ConntrackRecord = {
+  destination: string
+  destination_port?: string
+  destination_stats: {
+    bytes: string
+    packets: string
+  }
+  id: string
+  protocol: string
+  source: string
+  source_port?: string
+  source_stats: {
+    bytes: string
+    packets: string
+  }
+  state?: string
+  timeout: string
+}
+
+const isloading = ref(false)
+const ConntrackRecordsResponse = ref<ConntrackRecord[]>([])
+const error = ref({
+  notificationDescription: '',
+  notificationDetails: '',
+  notificationTitle: ''
+})
+const filter = ref('')
+const showDeleteModal = ref(false)
+
+onMounted(() => {
+  fetchConntrack()
+})
+
+const selectedItem = ref<ConntrackRecord>()
+
+async function fetchConntrack() {
+  error.value.notificationDescription = ''
+  error.value.notificationDetails = ''
+  try {
+    isloading.value = true
+    ConntrackRecordsResponse.value = (await ubusCall('ns.conntrack', 'list')).data.data
+  } catch (err: any) {
+    if (!(err instanceof CanceledError)) {
```

Comment:
If I'm not mistaken there is no way to cancel this API call, the `if` statement can be removed